app/templates/buyers/brief_publish_confirmation.html

```diff
@@ -56,7 +75,7 @@
           {% endif %}
         {% endfor %}
       {% endfor %}
-    {% else %}
+    {% elif not brief.lotSlug == 'digital-specialists' or brief.requirementsLength %}
```

Comment:
It's a minor point, but I'd go with `{% elif brief.lotSlug != 'digital-specialists' or brief.requirementsLength %}` to avoid the ambiguous `not` operator precedence